CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-### 2.4.1 (Next)
+### 2.5.0 (Next)
```

Comment:
Also need to increment version in version.rb.

Reply:
Ok, maybe 2.5 is too big a jump? Should I stick with 2.4.1?